### PR TITLE
Bump panel version to 1.3.0

### PR DESCRIPTION
**Pull request by philippjfr**
Simple version bump with some updated pins, including dropping of python 3.8 support.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,8 +26,7 @@ requirements:
     - wheel
     # These are also needed at build time.
     - bleach 4.1.0
-    - bokeh 3.1.1 # [py<39]
-    - bokeh 3.2.0 # [py>=39]
+    - bokeh 3.3.0
     - pyct 0.5.0
     - param 1.13.0
```
Comment (ryanskeith) [inline recipe/meta.yaml:31]: Is this correct?  It seems there is a mismatch between host and run requirements.

Comment (ryanskeith) [inline recipe/meta.yaml:31]:
```suggestion
    - param 2.0.0
```

**Review by ryanskeith — COMMENTED** [review]
There may be a problem with the host section.

**Review by ryanskeith — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM, but for next time please provide some description

**Pull request merged**